### PR TITLE
RDBC-267 in class Raven.Client.Documents.Session.DocumentsChanges som…

### PR DESCRIPTION
**Pull request by ml054**
…e fields are not used - marked FieldOldType & FieldNewType as obsolete

### CODE REVIEW

**Review by arekpalinski — APPROVED** [review]

File: src/Raven.Client/Documents/Session/DocumentsChanges.cs [hunk 1]
```diff
@@ -17,11 +18,13 @@ public class DocumentsChanges
         /// <summary>
         /// Previous field type.
         /// </summary>
+        [Obsolete("No longer used")]
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Session/DocumentsChanges.cs]: `[Obsolete("DocumentsChanges.FieldOldType is not supported anymore. Will be removed in next major version of the product.")]`

File: src/Raven.Client/Documents/Session/DocumentsChanges.cs [hunk 2]
```diff
@@ -17,11 +18,13 @@ public class DocumentsChanges
...
         public BlittableJsonToken FieldOldType { get; set; }
 
         /// <summary>
         /// Current field type.
         /// </summary>
+        [Obsolete("No longer used")]
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Session/DocumentsChanges.cs]: `[Obsolete("DocumentsChanges.FieldNewType is not supported anymore. Will be removed in next major version of the product.")]`

File: src/Raven.Client/Documents/Session/DocumentsChanges.cs (hunk 1 above)
Comment (ml054) [reply src/Raven.Client/Documents/Session/DocumentsChanges.cs]: done

File: src/Raven.Client/Documents/Session/DocumentsChanges.cs (hunk 2 above)
Comment (ml054) [reply src/Raven.Client/Documents/Session/DocumentsChanges.cs]: done

**Comment by ml054** [comment]
all things done - please recheck.

**Pull request merged**